.github/workflows/secure-deployment.yml

```
@@ -299,7 +299,7 @@ jobs:
   notify-success:
     needs: [deploy-staging, deploy-production]
     runs-on: ubuntu-latest
-    if: always() && (success() || failure())
+    if: always() && (success() || failure()) && secrets.SLACK_WEBHOOK != ''
```

actionlint (1.7.11)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)